src/types.ts

```diff
@@ -23,6 +23,10 @@ export type FormattedTab = Awaited<
 export type PublicCurrencyDetails = Pick<Currency, "isoCode" | "baseUnit">;
 
 export type UiConfig = {
+  paygateCopy: {
+    title: string;
+    moreOptionsCTA: string;
+  };
```

Comment:
We should also consider i18n as discussed yesterday, right? What would you propose for a structure? I'm thinking our options are:

```typescript
paygateCopy: {
  title: {
    en: string;
    es: string;
  }
}
```

or 

```typescript
paygateCopy: {
  en: {
    title: string;
    moreOptionsCTA: string;
  }
  es: {
    title: string;
    moreOptionsCTA: string;
}
```

Which would you prefer? Do you have any more ideas?

Reply:
I like the later one, are we only supporting this two languages for now?